modules/the-open-data-hub-user-interface.adoc

```diff
@@ -82,6 +82,11 @@ Settings -> Cluster settings::  The *Cluster settings* page allows you perform t
 * Reduce resource usage in your {productname-short} deployment by stopping notebook servers that have been idle.
 * Schedule notebook pods on tainted nodes by adding tolerations.
 
+Settings -> Accelerator profiles:: The *Accelerator profiles* page allows you perform the following administrative tasks on your accelerator profiles:
+* Enable or disable an existing accelerator profile, on the row containing the relevant accelerator profile, click the toggle in the *Enable* column.
```

Comment:
For the following two bullets, we don't describe how to carry out the task. Should we remove _, on the row containing the relevant accelerator profile, click the toggle in the *Enable* column_ for consistency?